src/main/resources/assets/api-docs.yaml

```diff
@@ -391,6 +419,13 @@ definitions:
           Optional list of reasons why a match result may have failed. Only supported in
           in versions 2 and higher
 
+  TranslateParagraph:
+    type: object
+    properties:
+      paragraph:
+        description: The paragraph to translate
+        type: string
```

Comment:
This is a great place for an example. Also, if we had a class for this, we could deserialize the JSON with that instead of the more generic form of:
```
final Map<String, String> body = mapper.readValue(jsonString, new TypeReference<>() {});
```
... and then we would not have to assume that `String paragraph = body.get("paragraph");` would be a valid call, we could instead just say:
```
TranslateParagraph payload = new Gson().fromJson(jsonString, TranslateParagraph.class);
String paragraph = payload.getParagraph();
```